x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/data/Block.java

```diff
@@ -281,4 +284,20 @@ static List<NamedWriteableRegistry.Entry> getNamedWriteables() {
             ConstantNullBlock.ENTRY
         );
     }
+
+    static Releasable releaseByDecRef(Block b) {
+        return () -> {
+            if (b != null) {
+                b.decRef();
+            }
+        };
+    }
+
+    static Releasable[] releaseByDecRef(Block... blocks) {
```

Comment:
I am not sure if we need this. I think I prefer Blocks can be closed multiple times (see the above comment).